specs/latest/2.0/index.html

```diff
@@ -2198,6 +2190,10 @@ <h2 class="no-toc">Table of contents</h2>
 
     <h3>New Features Supported in the WebGL 2 API</h3>
 
+    <div class="note editor">
+        Needs many updates for OpenGL ES 3.0 features
+    </div>
```

Comment:
I don't think that this note is needed - this is under "4 Other differences between WebGL 2.0 and WebGL 1.0", so this section does not need to cover anything that's covered by the above API documentation, and section 4 already has a similar comment at the top.